Signal/Images.xcassets/ic_speaker_bluetooth_inactive_audio_mode.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "ic_speaker_bluetooth_inactive_audio_mode.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
```

Comment:
Where'd you get this asset?  Also, don't forget to update https://github.com/WhisperSystems/Signal-iOS-Design.  I've been continuing to push changes to a branch https://github.com/WhisperSystems/Signal-iOS-Design/tree/charlesmchen/addIcons but I'm thinking of just making changes in master.